PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,42 @@
-<!-- Your title above should be a short description of what
-was changed. Do not include the issue number in the title. -->
+<!--
+Thank you so much for your PR!  To help us review your contribution, please
+consider the following points:
+
+- Your title above should be a short description of what was changed. Do not
+  include the issue number in the title.
+
+- Development information is available at
+  https://github.com/sympy/sympy/wiki/Development-workflow.
+
+- Do not create the PR out of master, but out of a separate branch.
```

Comment:
People who do this are so new to git that they likely aren't going to even understand this sentence. Maybe we can have a section on the wiki that goes into this in more detail.

Reply:
Yes, we'd need to link to some instructions on how to do this:

1. fork sympy on github
2. clone your fork
3. checkout a new branch
4. make your edits
5. commit
6. push your branch to your fork
7. open a pull request

It's surprisingly how non-trivial this is for new people. If you want to track master you also have to:

8. setup a remove to the main sympy source
9. merge in master into your branch

Reply:
What is missing in https://github.com/sympy/sympy/wiki/Development-workflow ?